crates/ty_python_semantic/resources/mdtest/call/union.md

```diff
@@ -227,17 +227,22 @@ def _(literals_2: Literal[0, 1], b: bool, flag: bool):
     literals_16 = 4 * literals_4 + literals_4  # Literal[0, 1, .., 15]
     literals_64 = 4 * literals_16 + literals_4  # Literal[0, 1, .., 63]
     literals_128 = 2 * literals_64 + literals_2  # Literal[0, 1, .., 127]
+    literals_256 = 2 * literals_128 + literals_2  # Literal[0, 1, .., 255]
 
-    # Going beyond the MAX_UNION_LITERALS limit (currently 200):
-    literals_256 = 16 * literals_16 + literals_16
-    reveal_type(literals_256)  # revealed: int
+    # Going beyond the MAX_UNION_LITERALS limit (currently 512):
+    literals_512 = 2 * literals_256 + literals_2  # Literal[0, 1, .., 511]
+    reveal_type(literals_512 if flag else 512)  # revealed: int
```

Comment:
Why do we need to add the `if flag else 512` here? It wasn't in the previous version of the test, and it seems the test passes without it. Did some intermediate version of the code fail without this?